src/__tests__/expected/dontWipeChrome.txt

```diff
@@ -6,37 +6,37 @@
  BBBBBB                                                  GR                                                                                                                                              
  index.html                                         | 11 ++-|    * [A] toggle selection of all (unique) files
  __________                                              GGR                                                                                                                                             
- scripts/makeDist.sh                                |  2 +-|    * [down arrow|j] move downward by 1
+ scripts/makeDist.sh                                |  2 +-|    * [d] describe file
  ___________________                                     GR                                                                                                                                              
- .../expected/selectCommandWithPassedCommand.txt    | 30 +++++++|    * [up arrow|k] move upward by 1
+ .../expected/selectCommandWithPassedCommand.txt    | 30 +++++++|    * [down arrow|j] move downward by 1
  _______________________________________________         GGGGGGG                                                                                                                                         
- src/__tests__/expected/selectDownSelectInverse.txt |  4 +-|    * [<space>] page down
+ src/__tests__/expected/selectDownSelectInverse.txt |  4 +-|    * [up arrow|k] move upward by 1
```

Comment:
the diff here probably would have been cleaner if you added this command to the bottom rather than in the middle. you can see the new `X` command was added last which made some of this a bit easer to digest :P